mantle/cmd/kola/switchkernel.go

```diff
@@ -90,7 +90,10 @@ var (
 
 func init() {
 	cmdSwitchKernel.Flags().StringVar(&rtKernelRpmDir, "kernel-rt", "", "Path to kernel rt rpm directory")
-	cmdSwitchKernel.MarkFlagRequired("kernel-rt")
+	err := cmdSwitchKernel.MarkFlagRequired("kernel-rt")
+	if err != nil {
+		fmt.Printf("failed to add kernel-rt as required flag: %v", err)
```

Comment:
```suggestion
if err != nil {
		panic(err)
```

I don't think this bit actually *can* fail in practice, and if it does somehow it should probably be fatal.  Printing errors and otherwise ignoring them is (in my opinion) a rather rare case.

Reply:
Thank you for review. That makes sense. Fixed.